README.md

```diff
@@ -3,253 +3,178 @@
-media.transcode('path/to/new_movie.flv', options)
-```
-
-### Specifying Input Options
-
-To specify which options apply the input, such as changing the input framerate, use `input_options` hash
-in the transcoder kwargs.
-
-```ruby
-movie = FFMPEG::Media.new('path/to/movie.mov')
-
-kwargs = { input_options: { framerate: '1/5' } }
-movie.transcode('path/to/new_movie.mp4', {}, **kwargs)
-
-# FFMPEG Command will look like this:
-# ffmpeg -y -framerate 1/5 -i path/to/movie.mov movie.mp4
-```
-
-### Watermarking
-
-Add watermark image on the video.
-
-For example, you want to add a watermark on the video at right top corner with 10px padding.
-
-```ruby
-options = {
-  watermark: 'path/to/watermark.png', resolution: '640x360',
-  watermark_filter: { position: 'RT', padding_x: 10, padding_y: 10 }
-}
-```
-
-Position can be "LT" (Left Top Corner), "RT" (Right Top Corner), "LB" (Left Bottom Corner), "RB" (Right Bottom Corner).
-The watermark will not appear unless `watermark_filter` specifies the position. `padding_x` and `padding_y` default to
-`10`.
-
-### Taking Screenshots
-
-You can use the screenshot method to make taking screenshots a bit simpler.
-
-```ruby
-media.screenshot('path/to/new_screenshot.jpg')
-```
-
-The screenshot method has the very same API as transcode so the same options will work.
+## Usage
 
-```ruby
-media.screenshot('path/to/new_screenshot.bmp', { seek_time: 5, resolution: '320x240' })
-```
+NOTE: For advanced usage, don't be afraid to dive into the source code.
+The gem is designed with care and documented all over the place, so dig in.
 
-To generate multiple screenshots in a single pass, specify `vframes` and a wildcard filename. Make
-sure to disable output file validation. The following code generates up to 20 screenshots every 10 seconds:
+### Metadata
 
 ```ruby
-media.screenshot('path/to/new_screenshot_%d.jpg', { vframes: 20, frame_rate: '1/6' }, validate: false)
-```
-
-To specify the quality when generating compressed screenshots (.jpg), use `quality` which specifies
-ffmpeg `-v:q` option. Quality is an integer between 1 and 31, where lower is better quality:
+media = FFMPEG::Media.new('path/to/media.mp4')
 
-```ruby
-media.screenshot('path/to/new_screenshot_%d.jpg', { quality: 3 })
-```
+media.valid? # true (would be false if ffmpeg fails to read the media metadata)
+media.local? # true (would be false if the file is remote)
+media.remote? # false
 
-You can preserve aspect ratio the same way as when using transcode.
+media.streams # [FFMPEG::Stream, FFMPEG::Stream]
+media.video_streams # [FFMPEG::Stream]
+media.video_streams? # true (tells you if the media has video streams or not)
+media.default_video_stream # FFMPEG::Stream
+media.audio_streams # [FFMPEG::Stream]
+media.audio_streams? # true (tells you if the media has audio streams or not)
+media.default_audio_stream # FFMPEG::Stream
+media.video? # true (tells you if the media has a movie stream – non-attached-picture)
```

Comment:
instead of `movie stream`, `video stream` would be more accurate I think

Reply:
unfortunately no, because a still image is a video stream in ffmpeg (an attached picture's codec type is video)

Reply:
I see, that's a bummer then